sympy/polys/matrices/domainmatrix.py

```diff
@@ -3370,6 +3420,39 @@ def lu_solve(self, rhs):
         sol = self.rep.lu_solve(rhs.rep)
         return self.from_rep(sol)
 
+    def PLDUdecompositionFF(self):
```

Comment:
I think I would call this `fflu` which is the name used in other places or just `pldu`. Other DomainMatrix methods all have lower-case names.